mockito-core/src/test/java/org/mockito/internal/stubbing/defaultanswers/ReturnsEmptyValuesTest.java

```diff
@@ -176,6 +176,33 @@ public void should_return_empty_duration() throws Exception {
         assertEquals("seconds of empty " + fqcn, 0L, seconds);
     }
 
+    @Test
+    public void should_return_empty_sequenced_collection_on_java21() throws Exception {
+        Class<?> sequencedCollectionClass = getClassOrSkipTest("java.util.SequencedCollection");
```

Comment:
@mockito/developers I wonder if we should split this test for each JDK?

Reply:
Yes, we can use `Assume.assumeThat` for this: https://github.com/search?q=repo%3Amockito%2Fmockito+assumeThat&type=code However, let's do that in a follow-up PR to keep things manageable.

@BeomSeogKim do you mind picking that up next?